frontend/components/dashboard/enrolled-droplets-grid-client.tsx

```
@@ -24,26 +24,23 @@ export function EnrolledDropletsGridClient({
     if (sortKey) {
       const [field, direction] = sortKey.split(":");
       sorted.sort((a, b) => {
-        let ratingA = ratingsMap.get(a.id);
-        let ratingB = ratingsMap.get(b.id);
-        if (!ratingA) {
-          ratingA = 0;
-        }
-        if (!ratingB) {
-          ratingB = 0;
-        }
+        let ratingA = ratingsMap.get(a.id) || 0;
```

ESLint

[error] 'ratingA' is never reassigned. Use 'const' instead.

(prefer-const)